appserver/payara-appserver-modules/jaxrs-client-tracing/src/main/java/fish/payara/requesttracing/jaxrs/client/decorators/JaxrsInvocationBuilderDecorator.java

```diff
@@ -309,5 +317,23 @@ public Response method(String name, Entity<?> entity) {
     public <T> T method(String name, Entity<?> entity, GenericType<T> responseType) {
         return this.invocationBuilder.method(name, entity, responseType);
     }
-    
+
+    /**
+     * Instruments this InvocationBuilder instance with OpenTracing
+     */
+    private void instrumentInvocationBuilder() {
+        // Get the ServiceLocator and OpenTracing services
+        ServiceLocator serviceLocator = Globals.getDefaultBaseServiceLocator();
+        OpenTracingService openTracing = serviceLocator.getService(OpenTracingService.class);
```

Comment:
do we need to access the service at every call of `instrumentInvocationBuilder`?